### DISCUSSION TITLE
Nuxt Module

### DISCUSSION THREAD

**Discussion by raphaelbernhart** [orestbida/cookieconsent#362] [Show and tell]
Hi @orestbida, 
I have created a nuxt module for cookieconsent :)
### https://github.com/raphaelbernhart/nuxtjs-cookieconsent

I would love to see the module in the README 😄

If it would help I would also make a PR to do this!
Just waiting on your reply to finish this.

PS: README and other changes for the developer experience (in the module repo) will be added soon.

**Comment by tmlmt** [comment]
Hey @raphaelbernhart great that you took the initiative to create a Nuxt module around this as it would be my use case. 

Would be great if there were plans for a little bit more to make it reusable by people: first of all some documentation (how to add your module to a Nuxt project, how to then use it), then availability of the build in npm. Should we continue the discussion on your repo through Issues? (You do not have Discussions enabled there)

**Reply by raphaelbernhart** [reply]
Hi @tmlmt,

thanks for your interest in the module. As I had no need for the module in my latest projects I placed it on hold for now.
But if there is use I would be eager to continue the development.

I also enabled Discussions in the repo so we can continue there (https://github.com/raphaelbernhart/nuxtjs-cookieconsent/discussions/4).
